libsql-server/src/http/user/listen.rs

```diff
@@ -149,9 +151,29 @@ async fn listen_stream(
                 },
                 Err(BroadcastStreamRecvError::Lagged(n)) => {
                     LISTEN_EVENTS_DROPPED.increment(n as u64);
+                    warn!(
+                        namespace = %namespace,
+                        table = %table,
+                        dropped_events = n,
+                        "Lagged event in listen stream"
+                    );
                     yield AggregatorEvent::Error(LAGGED_MSG);
                 },
-                _ => {}
+                Err(e) => {
+                    warn!(
+                        namespace = %namespace,
+                        table = %table,
+                        error = %e,
+                            "Unexpected error in listen stream"
+                    );
+                },
+                _ => {
+                    debug!(
+                        namespace = %namespace,
+                        table = %table,
+                        "Filtered out message in listen stream"
+                    );
+                }
```

Comment:
this branch can never be reached because the other branches already cover all the patterns

Reply:
I thought `Err(BroadcastStreamRecvError::Lagged(n))` was only invoked when that lagged msg matched? Everything else fallsthrough to `_`?

Reply:
I suggest reverting these changes as a whole. Before the compiler would let us know in case any new errors (that could be added by changes somewhere else) went unhandled. In this way, we don't have this guarantee. 

Also, warning when events are dropped may make the drop problem worse, since one reason we may drop events is that the time it takes to serialize them is higher than the time we take to generate them.

My understanding is that they add no functional changes, but let me know if I've missed something.